client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -227,19 +282,32 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 													}
 												);
 												setModalOpen( false );
-												doAccept();
+												/**
+												 * Handle the primary modal action.
+												 * If it's an inquiry, redirect to the order page; otherwise, continue with the default dispute acceptance.
+												 */
+												if ( isInquiry( dispute ) ) {
+													viewOrder();
+												} else {
+													doAccept();
+												}
 											} }
 										>
-											{ __(
-												'Accept dispute',
-												'woocommerce-payments'
-											) }
+											{ isInquiry( dispute )
+												? __(
+														'View Order to Issue Refund',
```

Comment:
Minor suggestion: only capitalise the first word in the button label to match `Accept dispute`.
```suggestion
														'View order to issue refund',
```